rpm/SPECS/openresty.spec

```diff
@@ -163,11 +159,9 @@ mkdir -p %{buildroot}/etc/init.d
 # to silence the check-rpath error
 export QA_RPATHS=$[ 0x0002 ]
 
-
 %clean
 rm -rf %{buildroot}
 
-
```

Comment:
Such blank lines are actually intended. We use 2 blank lines to separate sections like `%clean` and `%post`.